homework_8/geometric_figure/Triangle.java

```diff
@@ -0,0 +1,46 @@
+package geometric_figure;
+
+public class Triangle implements GeometricFigure {
+    private double a;
+    private double b;
+    private double c;
+
+    public Triangle(double a, double b, double c) {
+        this.a = a;
+        this.b = b;
+        this.c = c;
+    }
+
+    public double getA() {
+        return a;
+    }
+
+    public void setA(double a) {
+        this.a = a;
+    }
+
+    public double getB() {
+        return b;
+    }
+
+    public void setB(double b) {
+        this.b = b;
+    }
+
+    public double getC() {
+        return c;
+    }
+
+    public void setC(double c) {
+        this.c = c;
+    }
+
+    public double Perimetr() {
```

Comment:
The name of a variable should always start with a lowercase letter!
And also rename `Perimetr` to `calculatePerimeter`